sssd_test_framework/utils/override.py

```diff
@@ -0,0 +1,375 @@
+"Managing local users and groups."
+
+from __future__ import annotations
+
+import jc
+from pytest_mh import MultihostHost, MultihostUtility
+from pytest_mh.cli import CLIBuilder, CLIBuilderArgs
+from pytest_mh.ssh import SSHLog
+
+__all__ = [
+    "OverrideUtils",
+    "OverrideUser",
+    "OverrideGroup",
+]
+
+
+class OverrideUtils(MultihostUtility[MultihostHost]):
+    """
+    Management of override users and groups, sss_override.
+
+    .. note::
+
+        All changes are automatically reverted when a test is finished.
+    """
+
+    def __init__(self, host: MultihostHost) -> None:
+        """
+        :rtype: object
+        :param host: Remote host instance.
+        :type host: MultihostHost
+        """
+        super().__init__(host)
+
+        self.cli: CLIBuilder = CLIBuilder(host.ssh)
+        self._users: list[str] = []
+        self._groups: list[str] = []
+
+    def teardown(self) -> None:
+        """
+        Teardown
+        """
+        super().teardown()
+
+    def user(self, name: str) -> OverrideUser:
+        """
+        Get override user object.
+
+        .. code-block:: python
+            :caption: Example usage
+
+            @pytest.mark.topology(KnownTopology.Client)
+            def test_example(client: Client):
+                # Create override
+                client.override.users('user1').add(name='o-user1', uid=10001, gid=10001, gecos="test")
```

Comment:
Use " instead of ', outside of comments this is automatically changed by the black tool.